### PR TITLE
Improved translation

### PR DESCRIPTION
**Pull request by mreduar**
Improved translation to a more natural language in Spanish.

### CODE REVIEW

File: es/index.html [hunk 1]
```diff
@@ -86,7 +75,7 @@ <h1>No pidas permiso para preguntar, sólo pregunta.</h1>
   <footer>
     Contenido prácticamente robado de <a href="https://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
     -
-    Traducción de <a href="https://github.com/danirod">@danirod</a>
+    Traducción de <a href="https://github.com/mreduar">@mreduar</a>
```
Comment (tulir) [inline es/index.html:78]: You can add yourself to the credits, but you shouldn't just replace it

**Review by tulir — COMMENTED** [review]
@danirod do you want to review this since you made the original?

File: es/index.html (hunk 1 above)
Comment (mtatko) [reply es/index.html:78]:
```suggestion
    Traducción de <a href="https://github.com/danirod">@danirod</a> & <a href="https://github.com/mreduar">@mreduar</a>
```

Comment (danirod) [reply es/index.html:78]: > @danirod do you want to review this since you made the original?

Taking a look

File: es/index.html [hunk 2]
```diff
@@ -14,68 +14,57 @@
 </head>
 <body>
   <main>
-    <h1>No pidas permiso para preguntar, sólo pregunta.</h1>
+    <h1>No preguntes por preguntar, sólo pregunta</h1>
```
Comment (danirod) [inline es/index.html:17]: There is a difference in the statement intention between "No pidas permiso para preguntar" (prev) and "No preguntes por preguntar" (proposed).

* No pidas permiso para preguntar = Do not ask for permission to write your question.
* No preguntes por preguntar = Do not make questions just for the sake of making questions.

I've taken a look at other translations and they share the current intention: it's not about asking questions, it is about asking for permission first. I don't understand how the new title improves things.

File: es/index.html [hunk 3]
```diff
@@ -14,68 +14,57 @@
...
-      También estás dejando fuera a otras personas de forma innecesaria.
-      Muchas veces respondo preguntas que tienen que ver con lenguajes o librerías que nunca he usado,
-      porque las respuestas son (de algún modo para una persona que desarrolle) de sentido común.
+      Estás pidiéndole a la gente que asuma responsabilidades. Estás cuestionando la confianza de la gente sobre sus habilidades.
+      También estás excluyendo innecesariamente a otras personas. A menudo respondo a preguntas relacionadas con lenguajes o bibliotecas que nunca he utilizado,
+      porque las respuestas son (en el sentido de un programador) de sentido común.
     </p>
     <p>
-      También puede ser visto como...
+      También puede ser visto como..
```
Comment (danirod) [inline es/index.html:47]: I think that this is actually a typo in the English version, which indeed is using two dots. However, the proper way to do an ellipsis in Spanish (and in other languages as well after checking that /fr, /de, /it do it this way too) is with three dots, so I do not understand how this sentence improves with this change.

```suggestion
      También puede ser visto como...
```

**Review by danirod — COMMENTED** [review]
Some of the parts of the PR feel like using synonyms just for the sake of using synonyms without an actual improvement, however overall I think that some of the sentences in my old translation could be more neutral (more like an es translation and not like an es-es translation).

However, I've raised the hand in regards to changing the title. Other translations to other languages seem to agree that the focus is on the permission, not in the questions themselves, and I think that the new title changes the emphasis in a way that could be confusing.

**Comment by mtatko** [comment]
@mreduar can you please look at the changes suggested? Thanks

**Comment by mreduar** [comment]
It's done :) Thanks for the feedback, I agree.

**Review by mtatko — APPROVED** [review]
If @danirod has no further feedback, then LGTM

**Review by danirod — APPROVED** [review]
LGTM as well. Thank you for improving the translation 😃

Edit: sorry, I used the "approve" feature of GitHub because of my habits, even if it is not my repository, but you get the idea :)

**Comment by mtatko** [comment]
> Edit: sorry, I used the "approve" feature of GitHub because of my habits, even if it is not my repository, but you get the idea :)
It's all right.

Ready for merge @tulir

**Pull request merged**